content/post/29-js-ipfs-pubsub.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2017-07-13
+url: 29-js-ipfs-pubsub
+title: Decentralized pubsub on the browser using js-ipfs
+author: Pedro Teixeira
+---
+
+Recently, js-IPFS introduced experimental support for pub-sub.
+
+While not striclty a necessary competence of IPFS (since IPFS is more concerned with transport and protocols for managing content-addressing and all the resulting datastructures (like a Merkle DAG, blockchain, and others)), fortunately, the pub-sub protocol ended up being incorporated into its stack.
```

Comment:
`s/striclty/strictly`

I like the way you put it in the video "IPFS stands for the InterPlantery FileSystem but it is much more than that, a complete stack for distributed applications".